### PR TITLE
#EE-16704 - part of jira, make <StatsWidget/> able to get TextButton …

### PR DESCRIPTION
**Pull request by alex5il**
…for link usage,

On the way make it support all other component as well

### 🔦 Summary
...

### ✅ Checklist
- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [x] Story
  - [x] API description
  - [ ] Other (explain)
- 🔬 Change is tested
  - [x] Component
  - [ ] Visual test

### CODE REVIEW

File: src/StatsWidget/StatsWidget.driver.js [hunk 1]
```diff
@@ -19,6 +21,10 @@ const statsWidgetDriverFactory = ({ element }) => {
   return {
     exists: () => !!element,
 
+    getSuffixElementByHook: dataHook => {;
```
Comment (argshook) [inline src/StatsWidget/StatsWidget.driver.js]:
```suggestion
    getSuffixElementByHook: dataHook => {
```

File: src/StatsWidget/StatsWidget.driver.js [hunk 2]
```diff
@@ -19,6 +21,10 @@ const statsWidgetDriverFactory = ({ element }) => {
...
+      return findByHook(getSuffix(), 'wubba').textContent;
```
Comment (argshook) [inline src/StatsWidget/StatsWidget.driver.js]: `'wubba'`?

Comment (alex5il) [reply src/StatsWidget/StatsWidget.driver.js]: Yep bad naming 😅 
Will change to something better

Comment (argshook) [reply src/StatsWidget/StatsWidget.driver.js]: i don't think any name should be here :)

Comment (alex5il) [reply src/StatsWidget/StatsWidget.driver.js]: Hey @argshook any update on this? :)

**Pull request merged**